### PR TITLE
Port mevent_801BAAC from fire red

### PR DESCRIPTION
**Pull request by PikalaxALT**
0 NONMATCHING

### CODE REVIEW

File: .idea/codeStyles/Project.xml [hunk 1]
```diff
@@ -0,0 +1,17 @@
+<component name="ProjectCodeStyleConfiguration">
```
Comment (ProjectRevoTPP) [inline .idea/codeStyles/Project.xml]: Did you mean to commit this?

Comment (PikalaxALT) [reply .idea/codeStyles/Project.xml]: no. tell jetbrains to fuck off for me please

File: src/mevent_801BAAC.c [hunk 2]
```diff
@@ -0,0 +1,804 @@
...
+
+extern const struct OamData gUnknown_08524934;
+
+const u8 gUnknown_082F0E10[][3] = {
+    {0, 2, 3},
+    {0, 1, 2}
+};
+const u8 ALIGNED(4) gUnknown_082F0E18[3] = {7, 4, 7};
+const struct WindowTemplate gUnknown_082F0E1C[] = {
+    {0x01, 0x01, 0x01, 0x19, 0x04, 0x02, 0x029c},
```
Comment (huderlem) [inline src/mevent_801BAAC.c]: Please use the syntax that explicitly names each field in the struct.

File: src/mevent_801BAAC.c [hunk 3]
```diff
@@ -0,0 +1,804 @@
...
+const u32 gWonderNewsGfx2[] = INCBIN_U32("graphics/wonder_transfers/wonder_news_2.4bpp.lz");
+const u32 gWonderNewsTilemap2[] = INCBIN_U32("graphics/wonder_transfers/wonder_news_2.bin.lz");
+const u32 gWonderNewsGfx3[] = INCBIN_U32("graphics/wonder_transfers/wonder_news_3.4bpp.lz");
+const u32 gWonderNewsTilemap3[] = INCBIN_U32("graphics/wonder_transfers/wonder_news_3.bin.lz");
+const u32 gWonderNewsGfx7[] = INCBIN_U32("graphics/wonder_transfers/wonder_news_7.4bpp.lz");
+const u32 gWonderNewsTilemap7[] = INCBIN_U32("graphics/wonder_transfers/wonder_news_7.bin.lz");
+const u32 gWonderNewsGfx8[] = INCBIN_U32("graphics/wonder_transfers/wonder_news_8.4bpp.lz");
+const u32 gWonderNewsTilemap8[] = INCBIN_U32("graphics/wonder_transfers/wonder_news_8.bin.lz");
+
+/*
```
Comment (huderlem) [inline src/mevent_801BAAC.c]: Remove this big comment block

**Pull request merged**